### ISSUE TITLE
Rogue - Hack and Slash

### ISSUE THREAD

**Issue by evilkill** [dalaranwow/dalaran-wow#3912]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote @http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Does not proc properly for special abilities. I did notice occasional white attacks that would happen after an SS that might have either been random errors with the way I was standing or how hack and slash procs for special abilities on this server (ironically this is what procced the only actual hack and slash attack). Either way this screenshot shot shows that hack and slash does not work properly (if the melee attacks are supposed to be the procs from specials then they are doing significantly less damage)

**Expected behaviour**: Tell us what should happen instead

Hack and Slash should proc off special abilities and do damage equal to the amount that the attack hit for.

**Steps to reproduce the problem**:

1.  Go dummy
2.  Stand at angle so auto attacks wont go off
3.  Spam SS/Evis

![capture](https://cloud.githubusercontent.com/assets/24283210/20778584/88ebf3a2-b73c-11e6-92ba-cdc2f805f02b.PNG)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

http://wowwiki.wikia.com/wiki/Hack_and_Slash

**UPDATE**

after more testing and double checking my position by attacking the dummy while in my mutilate spec and making sure i was getting 0 auto attacks it seems that hack and slash procs from SS and evis are being registered as auto attacks and not as hack and slash procs. while this may be better then nothing it is still doing less dmg on average then SS and evis (should be equal) and suffers from white miss chances and glancing blows (also shouldn't happen) reducing overall dps by more then the gain we get from the extra procs (the extra auto attack from the special attacks can themselves proc hack and slash)

screen shots:

![capture3](https://cloud.githubusercontent.com/assets/24283210/20779699/2250228c-b744-11e6-8adf-b9a82fb3cd88.PNG)
![capture4](https://cloud.githubusercontent.com/assets/24283210/20779700/253469b8-b744-11e6-91ec-f2f74c5ef3be.PNG)
![capture2](https://cloud.githubusercontent.com/assets/24283210/20779704/2ad1e120-b744-11e6-922c-9e90416d6d0b.PNG)

Screenshot showing no melee attacks with muti spec from the same positioning:
![capture5](https://cloud.githubusercontent.com/assets/24283210/20779938/d673e91e-b745-11e6-8338-b3c4580a22f3.PNG)

**Comment by evilkill** [comment]
Sorry for not tagging properly. if you could add the tags that would be appreciated [talent][rogue]

**Comment by Rushor** [comment]
moved to https://github.com/dalaranwow/dalaran-wow/issues/3617
